README.md

```diff
@@ -1,106 +1,130 @@
+const df = require("durable-functions");
+const { app } = require("@azure/functions");
+
+app.http("httpStart", {
+    route: "orchestrators/{orchestratorName}",
+    extraInputs: [df.input.durableClient()],
+    handler: async (request, context) => {
+        const client = df.getClient(context);
+        const body = await request.json();
+        const instanceId = await client.startNew(request.params.orchestratorName, { input: body });
 
-    context.log(`Started orchestration with ID = '${instanceId}'.`);
+        context.log(`Started orchestration with ID = '${instanceId}'.`);
 
-    return client.createCheckStatusResponse(context.bindingData.req, instanceId);
-};
+        return client.createCheckStatusResponse(request, instanceId);
+    },
+});
 ```
 
-**Note:** Client functions are started by a trigger binding available in the Azure Functions 2.x major version. [Read more about trigger bindings and 2.x-supported bindings.](https://docs.microsoft.com/en-us/azure/azure-functions/functions-triggers-bindings#overview)
+**Note:** Client functions can be started by any trigger binding supported in the Azure Functions runtime version 2.x+. Node.js v4 programming model apps require at least version 4.16.5 of the Azure Functions runtime. [Read more about trigger bindings and 2.x-supported bindings.](https://docs.microsoft.com/en-us/azure/azure-functions/functions-triggers-bindings#overview)
 
 ## Samples
 
-The [Durable Functions samples](https://docs.microsoft.com/en-us/azure/azure-functions/durable-functions-install) demonstrate several common use cases. They are located in the [samples directory.](./samples/) Descriptive documentation is also available:
+The [Durable Functions samples](https://docs.microsoft.com/en-us/azure/azure-functions/durable-functions-install) demonstrate several common use cases. They are located in the samples directories ([JavaScript](./samples-js/)/[TypeScript](./samples-ts/)). Descriptive documentation is also available:
 
-* [Function Chaining - Hello Sequence](https://docs.microsoft.com/en-us/azure/azure-functions/durable-functions-sequence)
-* [Fan-out/Fan-in - Cloud Backup](https://docs.microsoft.com/en-us/azure/azure-functions/durable-functions-cloud-backup)
-* [Monitors - Weather Watcher](https://docs.microsoft.com/en-us/azure/azure-functions/durable-functions-monitor)
-* [Human Interaction & Timeouts - Phone Verification](https://docs.microsoft.com/en-us/azure/azure-functions/durable-functions-phone-verification)
+-   [Function Chaining - Hello Sequence](https://docs.microsoft.com/en-us/azure/azure-functions/durable-functions-sequence?tabs=javascript-v4)
+-   [Fan-out/Fan-in - Cloud Backup](https://docs.microsoft.com/en-us/azure/azure-functions/durable-functions-cloud-backup?tabs=javascript-v4)
+-   [Monitors - Weather Watcher](https://docs.microsoft.com/en-us/azure/azure-functions/durable-functions-monitor?tabs=javascript)
```

Comment:
This sample doesn't have a V4 version yet (Because the whole API used is deprecated, and we'd have to make changes to the page as a whole not just the code snippets - so we punted this until after preview announcement and language-specific pivots)